spec/models/course_survey_response_map_spec.rb

```
@@ -27,7 +27,7 @@
     describe '#get_title' do
       it 'returns the correct title constant' do
         course_survey_response_map = CourseSurveyResponseMap.new
-        expect(course_survey_response_map.get_title).to eq(ResponseMapSubclassTitles::COURSE_SURVEY_RESPONSE_MAP_TITLE)
+        expect(course_survey_response_map.get_title).to eq(ExpertizaConstants::ResponseMapTitles::COURSE_SURVEY_RESPONSE_MAP_TITLE)
       end
     end
-  end
\ No newline at end of file
+  end
```

RuboCop (1.86.1)

[convention] `end` at 33, 2 is not aligned with `RSpec.describe CourseSurveyResponseMap, type: :model do` at 3, 0.

(Layout/BlockAlignment)